Paranormal/Paranormal/Panels/Preview/PreviewSpriteUtils.swift

```diff
@@ -53,4 +54,40 @@ class PreviewSpriteUtils: NSObject {
             return NSImage(CGImage: cgImage, size: sprite.contentSize)
         }
     }
+
+    class func grayImage(#width: UInt, height: UInt, brightness: Float) -> NSImage {
+        let colorSpace : CGColorSpace = CGColorSpaceCreateDeviceRGB()
+        let bitmapInfo = CGBitmapInfo(CGImageAlphaInfo.PremultipliedLast.rawValue)
+
+        var context = CGBitmapContextCreate(nil, width, height,
+            8, 0, colorSpace, bitmapInfo)
+        let color = CGColorCreateGenericRGB(
+            CGFloat(brightness), CGFloat(brightness), CGFloat(brightness), 1.0)
+        CGContextSetFillColorWithColor(context, color)
+        let rect = CGRectMake(0, 0, CGFloat(width), CGFloat(height))
+        CGContextFillRect(context, rect)
+        let cgImage = CGBitmapContextCreateImage(context)
+
+        let size = NSSize(width: CGFloat(width), height: CGFloat(height))
+        let grayNSImage = NSImage(CGImage: cgImage, size: size)
+        return grayNSImage
+    }
+
+    class func grayImageWithAlphaSource(source: NSImage, brightness: Float) -> NSImage {
```

Comment:
Could you throw a comment here that this function needs to be run on GPUImage thread? Could lead to miss leading in the future.